iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ZipLoadModel.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * ZIP (polynomial) load model.
```

Comment:
Same remark, if nominal voltage, I think it should be in the javadoc. I can do it!